packages/in_app_purchase/in_app_purchase/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 3.2.3
+* Bumps minimum `in_app_purchase_storekit` version to 0.4.0.
+* Bumps example `in_app_purchase_storekit` version to 0.4.0.
```

Comment:
We don't need to describe mechanical changes to the example app, so this line can go.